include/cpp2util.h

```diff
@@ -408,7 +408,7 @@ auto assert_in_bounds(auto&& x, auto&& arg CPP2_SOURCE_LOCATION_PARAM_WITH_DEFAU
 #endif
 }
 
-auto Uncaught_exceptions() -> int {
+inline auto Uncaught_exceptions() -> int {
```

Comment:
Can't add on the actual line in GH, but I think we need an inline in front of `Throw' L399 aswell.